### PR TITLE
Debian11 and log4shell patch

### LINKED ISSUE #44: [FEATURE] Add Dlog4j2.formatMsgNoLookups to prevent log4shell
### Description

Add the flag `Dlog4j2.formatMsgNoLookups=true` to avoid log4shell vulnerability

### Why is this needed?
Security concerns

### Additional Information

[Any additional information, configuration or data.]

### LINKED ISSUE #45: [FEATURE] Support for debian bullseye
### Description

[Description of the issue]

### Why is this needed?

[A clear and concise description of why do you think this is needed]

### Additional Information

[Any additional information, configuration or data.]

### PR DESCRIPTION
**Pull request by vicsufer**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Applicable Issues

Close #44 
Close #45

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
 
 ## [Unreleased](https://github.com/idealista/wildfly_role/tree/develop)
+### Added
+- *[#44](https://github.com/idealista/wildfly_role/issues/44) Add log4shell prevention flag by default* @vsuarez
```
Comment (ablancolopez) [inline CHANGELOG.md]: wrong username?

File: Pipfile [hunk 2]
```diff
@@ -12,4 +12,4 @@ molecule = "==3.0.4"
 docker = "==4.2.2"
```
Comment (ablancolopez) [inline Pipfile]: should pipfile deps be upgraded too?

**Review by ablancolopez — APPROVED** [review]

File: Pipfile (hunk 2 above)
Comment (vicsufer) [reply Pipfile]: Fixed

File: CHANGELOG.md (hunk 1 above)
Comment (vicsufer) [reply CHANGELOG.md]: Fixed

**Review by sorobon — APPROVED** [review]

**Pull request merged**